src/tag/tag-filter.component.ts

```diff
@@ -2,37 +2,59 @@ import {
 	Component,
 	Output,
 	EventEmitter,
-	HostBinding
+	HostBinding,
+	Input,
+	ViewChild,
+	ElementRef
 } from "@angular/core";
 import { Tag } from "./tag.component";
 
 @Component({
 	selector: "ibm-tag-filter",
 	template: `
-		<ng-content></ng-content>
-		<svg
-			(click)="close.emit()"
-			focusable="false"
-			preserveAspectRatio="xMidYMid meet"
+		<span #label class="bx--tag__label" [title]="label.innerHTML">
```

Comment:
We may want to use `textContent` for the title, since `innerHTML` may end up returning some weird HTML/SVG content ... We may also want to expose that as an Input for better control

Reply:
Hey! Yes, I'm open to your preference/suggestion. I was trying to find an analog for how the React library is doing it and this was the best I came up with, lol.

My preference is for it to be an Input. Would we keep `<ng-content />` in that case, to allow for complex content, or would we replace it with the Input's value?

Reply:
I'm just realizing that for now we have to keep `<ng-content />` or else it's a breaking change... BUT, I would recommend discussing w/the team whether in future versions you want to allow more complex tag-filter content than a simple title/string.

Reply:
Now exposed as an input: `title`